.github/workflows/pytest.yml

```diff
@@ -0,0 +1,31 @@
+name: Pytest
+
+on: [push]
+
+jobs:
+  run-pytest:
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:13.2
+        env:
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: copy_cat
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install -e .
+      - name: Test with pytest
+        run: python -m pytest
```

Comment:
Uses `python -m pytest` instead of `pytest` 

> Actually pytest and python -m pytest are almost the same except for one difference: the latter will put the current directory into sys.path. This is standard Python behavior, because you are executing python, not the pytest executable. This is the reason why pytest -m pytest works, pets is at your repository root and ends up being found by python.

https://github.com/pytest-dev/pytest/issues/2287